src/main/webapp/creatives/maps/simid_map_creative.js

```diff
@@ -50,6 +52,37 @@ export default class SimidMapCreative extends BaseSimidCreative {
    * @private 
   */
   grantLocationAccess_() {
-    //ToDo(kristenmason@): implement map
+    //ToDo(kristenmason@): implement the Google Maps request access functionality
+    this.simidProtocol.sendMessage(CreativeMessage.REQUEST_PAUSE).then(() => {
+      findNearest.classList.add("hidden");
+      //ToDo(kristenmason@): display map
+      this.createButtons_();
+    })
+  }
+
+  createButtons_() {
```

Comment:
createButtons is kind of a vague name.  I think this is for specifically buttons when the map is being accessed. If that's the case, either comment this function or better yet rename the function.

Reply:
I added jsdoc explaining the function in the second commit (maybe you didn't see it). I can change the name also